.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,8 @@
 <!-- 🚨 Please Do Not skip any instructions and information mentioned below as they are all required and essential to evaluate and test the PR. By fulfilling all the required information you will be able to reduce the volume of questions and most likely help merge the PR faster 🚨 -->
 
+<!-- It is essential to keep the **☑️ Allow edits by maintainers** checked in Pull Request Template as it increases collaboration by permitting commits to the PR branch created from a fork  -->
```

Comment:
```suggestion
<!-- 📝 It is preferred if you keep the "☑️ Allow edits by maintainers" checked in the Pull Request Template as it increases collaboration with the Toolkit maintainers by permitting commits to your PR branch (only) created from your fork.  This can let us quickly make fixes for minor typos or forgotten StyleCop issues during review without needing to wait on you doing extra work. Let us help you help us! 🎉 -->
```

Thoughts on being a bit more detailed on the reasoning here? (And also subtle reminders for things to check before submitting PR. 😋) Also converted the '**' to quotes as the markdown won't be seen in the editor as it's a comment.

Reply:
I think this should suffice 🚀